_posts/2020-02-26-flink-for-data-warehouse.md

```diff
@@ -0,0 +1,108 @@
+---
+layout: post 
+title:  "Flink as Unified Engine for Modern Datawarehousing: Production Ready Hive Integration" 
+date: 2020-02-26T02:30:00.000Z
+categories: news
+authors:
+- Bowen Li
+- twitter: Bowen__Li
+---
+
+In this blog post, you will learn how Flink can help modernize your data warehouse and our motivation behind the Flink-Hive integration, what you can currently do with it in Flink's latest release 1.10, and our roadmap for Flink 1.11 and beyond.
+
+{% toc %}
+
+
+## Introduction 
+
+What are some of the latest requirements for your data warehouse and data infrastructure in 2020?
+
+We’ve came up with some for you.
+
+Firstly, today’s business is shifting to a more real-time fashion, and thus demands abilities to process online streaming data with low latency for near-real-time or even real-time analytics. People become less and less tolerant of delays between when data is generated and when it arrives at their hands, ready to use. Hours or even days of delay is not acceptable anymore. Users are expecting minutes, or even seconds, of end-to-end latency for data in their warehouse, to get quicker-than-ever insights.
+
+Secondly, the infrastructure should be able to handle both offline batch data for offline analytics and exploration, and online streaming data for more timely analytics. Both are indispensable as they both have very valid use cases. Despite real time processing mentioned above, batch would still exist as it’s good for ad hoc queries and explorations, and full size calculations. Your modern infrastructure should not force users to choose between one or the other, it should offer users both options for a world-class data infrastructure.
```

Comment:
You emphasize the importance of streaming analytics, but Hive and most of the features mention in the post focus on batch processing. I think it would be a good idea to explain how Flink connects the Hive-integration features with streaming analytics.

Reply:
Basically tell the reader what's possible with Flink + Hive-Integration that's not possible with Hive

Reply:
good catch! added a section "streaming processing"